packages/terra-form/src/components/Input.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+
+const propTypes = {
+  /**
+   * The defaultValue of the input field. Use this to create an uncontrolled input.
+   */
+  defaultValue: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+  ]),
+  /**
+   * Name of the input
+   */
+  name: PropTypes.string,
+  /**
+   * Functional to be called when the input is changed.
+   */
+  onChange: PropTypes.func,
```

Comment:
In my opinion, I think we should keep just onChange as first class, as that's the most critical one for dealing with controlled inputs. There are several others we could add, but that could make this list very large.